### PR TITLE
[Auctions] Adds responsiveness and iPad support to layout

### PR DESCRIPTION
**Pull request by l2succes**
This includes various UI tweaks and fixes
 - The text on the top tab bar is now centered correctly
 - Added spacing between the title and live icon on auction item cards 
 - "Register by" is now on its own line

iPad 

iPhone 6 Plus

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,9 @@
 
 -   Adds for For You tab with styled headers - luc
 -   Creates tab view container for new home - luc
+-   Adds Live and timed auctions in home view - luc
+-   Tab bar text now aligned and centered correctly - luc
+-   Adds iPad support to layout - luc
```
Comment (orta) [inline CHANGELOG.md]: > Adds iPad support to layout 

Doesn't say to which screen

File: src/lib/Scenes/Home/Components/Sales/Components/SaleItem.tsx [hunk 2]
```diff
@@ -2,17 +2,23 @@ import moment from "moment"
...
 import fonts from "../../../../../../data/fonts"
 import OpaqueImageView from "../../../../../Components/OpaqueImageView"
 import Serif from "../../../../../Components/Text/Serif"
 import { liveDate, timedDate } from "../formatDate"
 
+const screenSize = Dimensions.get("window")
+const isIPad = screenSize.width > 700
+const numColumns = isIPad ? 4 : 2
+const gutterSize = isIPad ? 100 : 60
+const containerWidth = (screenSize.width - gutterSize) / numColumns
```
Comment (orta) [inline src/lib/Scenes/Home/Components/Sales/Components/SaleItem.tsx]: these can change at runtime (rotation, iOS window resizing), so I'd recommend putting them in render

Comment (l2succes) [reply src/lib/Scenes/Home/Components/Sales/Components/SaleItem.tsx]: Good point

File: CHANGELOG.md (hunk 1 above)
Comment (l2succes) [reply CHANGELOG.md]: 😅 silly me, thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -39,6 +39,9 @@
 
 -   Adds for For You tab with styled headers - luc
 -   Creates tab view container for new home - luc
+-   Adds Live and timed auctions in home view - luc
+-   Tab bar text now aligned and centered correctly - luc
+-   Adds iPad support for Auctions view layout - luc
```
Comment (orta) [inline CHANGELOG.md]: Ah - these are all on the wrong section also, I'll merge this ( as it's not exactly a PR blocker ) and edit the CHANGELOG on my next PR

**Pull request merged**